pilot/docker/Dockerfile.proxyv2

```diff
@@ -49,7 +49,9 @@ COPY pilot-agent /usr/local/bin/pilot-agent
 COPY envoy_policy.yaml.tmpl /var/lib/istio/envoy/envoy_policy.yaml.tmpl
 
 COPY stats-filter.wasm /etc/istio/extensions/stats-filter.wasm
```

Comment:
Should we just copy the pre complied ones? Original Wasm file is not used anywhere.

Reply:
What's the size of these files btw?

Reply:
5M for mx, 1.5 for stats.

Reply:
I guess protobuf is 3.5M compiled :/

Reply:
Keeping both in so we can continue to test easily after code freeze.
Will remove plain files later.